pandas/tests/io/test_sql.py

```diff
@@ -2819,301 +2795,7 @@ def clean_up(test_table_to_drop):
             index=False,
         )
         assert tquery(sql_select, con=self.conn) == [(3, "C"), (4, "D"), (5, "E")]
-        clean_up(table_name)
-
-        # test if_exists='append'
-        sql.to_sql(
-            frame=df_if_exists_1,
-            con=self.conn,
-            name=table_name,
-            if_exists="fail",
-            index=False,
-        )
-        assert tquery(sql_select, con=self.conn) == [(1, "A"), (2, "B")]
-        sql.to_sql(
-            frame=df_if_exists_2,
-            con=self.conn,
-            name=table_name,
-            if_exists="append",
-            index=False,
-        )
-        assert tquery(sql_select, con=self.conn) == [
-            (1, "A"),
-            (2, "B"),
-            (3, "C"),
-            (4, "D"),
-            (5, "E"),
-        ]
-        clean_up(table_name)
-
-
-@pytest.mark.single
-@pytest.mark.db
-@pytest.mark.skip(
-    reason="gh-13611: there is no support for MySQL if SQLAlchemy is not installed"
```

Comment:
Just curious, is is not possible for us to test MySQL with these test or are these covered in different tests?

Reply:
These are covered in different tests using sqlalchemy. My guess is many years ago pandas's SQL functions were implemented directly on top of SQL drivers like pymysql. And after refactoring using sqlalchemy, those old tests were skipped but not removed for some reason.